models/models.go

```diff
@@ -176,6 +178,22 @@ func (r *Route) loadUser(db *gorm.DB) (utils.User, error) {
 	return LoadUser(userData)
 }
 
+func (r *Route) GetHeaders() (headers utils.Headers) {
+	headers = utils.Headers{}
+	for _, header := range r.Headers {
+		headers.Add(header.Header)
+	}
+	return
+}
+
+func (r *Route) SetHeaders(headers utils.Headers) {
+	routeHeaders := []RouteHeader{}
```

Comment:
This should be either `var routeHeaders []RouteHeader` or `routeHeaders := make([]RouteHeader, 0)`. Leaving it like this is considered an empty declaration instead of a nil value and the first value will be empty with no values, making this a potential bug. :)